app/assets/stylesheets/components/about.scss

```diff
@@ -0,0 +1,198 @@
+#about {
+
+    .row {
+        padding: 40px;
+        color: $text-primary;
+    }
+    
+    .intro {
+        text-align: center;
```

Comment:
karenfreemansmith ha, looks like you changed the style definition.

But what I meant was that, instead of  defining `text-align: center`, you could use the bootstrap class `text-center`